src/reducers/notification.js

```diff
@@ -6,8 +6,11 @@ export const initialState = {
 
 const defaultNotificationState = {
```

Comment:
Absolutely no need for this object. `id`, `title` are specified in the action creator, other fields are optional and could be applied directly without runtime overhead for `...defaultNotificationState`